packages/common/src/interceptors/geoip.interceptor.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Injectable,
+  NestInterceptor,
+  ExecutionContext,
+  CallHandler,
+  Logger,
+  HttpStatus,
+  InternalServerErrorException,
+  HttpException,
+} from '@nestjs/common';
+import { Observable } from 'rxjs';
+import { HttpService } from '@nestjs/axios';
+
+@Injectable()
+export class GeoIPInterceptor implements NestInterceptor {
+  private readonly httpService: HttpService;
+  private readonly allowedCountries: string[];
+
```

Comment:
register a logger attribute as well
```suggestion
private readonly logger: Logger ;
```